README.rst

```diff
@@ -29,5 +29,5 @@ Disclaimer
 **********
 
 nRF Connect SDK supports development with nRF9160 Cellular IoT devices.
-nRF53 Series devices (which are pre-production) and Zigbee and Bluetooth mesh protocols are supported for development in v1.4.2 for prototyping and evaluation.
-Support for production and deployment in end products is coming soon.
+nRF53 Series devices are now supported for production.
```

Comment:
I believe the whole "Disclaimer" section can be remove.  This only served a purpose while nRF53 was prototype.  This section is not required.

Reply:
Removed the section.